### PR TITLE
Hook up repo to CoreOS CI and run pylint

### PR DESCRIPTION
**Pull request by jlebon**
Let's do a pylint run on the repo so we can catch basic stuff like
missing variables or functions, etc...

### CODE REVIEW

**Comment by jlebon** [comment]
Requires https://github.com/coreos/coreos-assembler/pull/1384 for pylint.

**Comment by jlebon** [comment]
@dustymabe Want to take another look at the final diff?

File: .cci.jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// Documentation: https://github.com/coreos/coreos-ci/blob/master/README-upstream-ci.md
+
+cosaPod(runAsUser: 0) {
+    checkout scm
+    // XXX: should have a deps.txt file for each subproject or something
+    shwrap("dnf install -y python3-aiohttp")
```
Comment (dustymabe) [inline .cci.jenkinsfile:6]: can we say which app requires this?

Comment (jlebon) [reply .cci.jenkinsfile:7]: Done!

**Pull request merged**